cypress/integration/control_attach.js

```diff
@@ -0,0 +1,89 @@
+context('Date Control', () => {
+	before(() => {
+		cy.login();
+		cy.visit('/app/doctype');
+		return cy.window().its('frappe').then(frappe => {
+			return frappe.xcall('frappe.tests.ui_test_helpers.create_doctype', {
+				name: 'Test Attach Control',
+				fields: [
+					{
+						"label": "Attach File or Image",
+						"fieldname": "attach",
+						"fieldtype": "Attach",
+						"in_list_view": 1,
+					},
+				]
+			});
+		});
+	});
+	it('Checking functionality for "Link" button in the "Attach" fieldtype', () => {
+		//Navigating to the new form for the newly created doctype
+		cy.new_form('Test Attach Control');	
+
+		//Clicking on the attach button which is displayed as part of creating a doctype with "Attach" fieldtype
+		cy.get('.control-input > .btn-sm').contains('Attach').click();
+
+		//Clicking on "Link" button to attach a file using the "Link" button
+		cy.get('.mt-2').findByRole('button', {name: 'Link'}).click();
+		cy.get('.input-group > .form-control').type('https://wallpaperplay.com/walls/full/8/2/b/72402.jpg');
```

Comment:
which frappe control are you selecting?

Reply:
its a field wherein I am entering the link of the image I want to upload...I have corrected the selector using "findByPlaceholderText" function.